packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -482,10 +480,10 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   }
 
   // Set during layout if overflow occurred on the main axis.
-  double _overflow;
+  double? _overflow;
```

Comment:
this should be non-nullable and late, there's no valid reason to access this before layout iirc (such accesses could be guarded with something like hasSize)

Reply:
It is nullable because there is a null check in:

```dart
  String toStringShort() {
    String header = super.toStringShort();
    if (_overflow is double && _hasOverflow)
      header += ' OVERFLOWING';
    return header;
  }
```

BTW testing nullability with `is double` instead of `!= null` looks really awful. (I will change it)

Reply:
hm, yeah, if we expose it in the debug stuff then fair enough.